resources/assets/styles/main.css

```diff
@@ -32,22 +37,6 @@
   @apply text-xs;
 }
 
-.min-h-10vh {
```

Comment:
i'm moving them to the `tailwind.config.js` file, the reason being that in this case those classes won't automatically get variants like: `sm:min-h-10vh; 2xl:min-h-15vh`